incidentresponse/customresponse/slack/event_handlers.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from urllib.parse import urljoin
+
+from django.urls import reverse
+from django.conf import settings
+from response.slack.decorators import (
+    slack_event,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@slack_event("member_joined_channel")
```

Comment:
I guess this only acts on response channels? I tried joining other channels and it did nothing

Reply:
Yes, it only triggers on comms channels ( which have an incident associated to it in response DB) .